site/docs/4.3/examples/product/index.html

```diff
@@ -7,7 +7,7 @@
 <nav class="site-header sticky-top py-1">
   <div class="container d-flex flex-column flex-md-row justify-content-between">
     <a class="py-2" href="#">
-      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="d-block mx-auto" role="img" viewBox="0 0 24 24" focusable="false"><title>Product</title><circle cx="12" cy="12" r="10"/><path d="M14.31 8l5.74 9.94M9.69 8h11.48M7.38 12l5.74-9.94M9.69 16L3.95 6.06M14.31 16H2.83m13.79-4l-5.74 9.94"/></svg>
+      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" class="d-block mx-auto" role="img" viewBox="0 0 24 24" focusable="false" aria-labelledby="svg-product-title"><title id="svg-product-title">Product</title><circle cx="12" cy="12" r="10"/><path d="M14.31 8l5.74 9.94M9.69 8h11.48M7.38 12l5.74-9.94M9.69 16L3.95 6.06M14.31 16H2.83m13.79-4l-5.74 9.94"/></svg>
```

Comment:
advantage of this over just adding an `aria-label="..."` on the link?

Reply:
I've not tested with aria-label. I will made a codepen.
I've propose that solution based on https://developer.paciellogroup.com/blog/2013/12/using-aria-enhance-svg-accessibility/ but that doesn't discuss about svg used in a link

Reply:
First tests give the advantage to aria-label on the link that prevent to vocalize the type of the content, but impose to duplicate the title (no impact in our case)
test NVDA 2016.2.1 / chrome 72 : test NVDA 2016.2.1 / firefox ESR 52.7.2 : test Jaws 17 / IE11 : Same result
labelledby on the SVG say : Search graphique lien (in french)
aria-label on the link say : Search lien (in french)

If you've more feedback with voiceover/talkback/narrator let me know

Reply:
anyone had made some more test?
What solution have your favour @patrickhlauke ? labelledby or aria-label? Let me know if I need to update the PR